index.html

```diff
@@ -72,9 +129,258 @@ <h2 class="greeting">
+          </p>
+        </div>
+        <div>
+          <ul class="list-of-course">
+                <li><a href="#">html</a></li>
+                <li><a href="#">css</a></li>
+                <li><a href="#">Javascript</a></li>
+          </ul>
+        </div>
+        <div >
+          <a  class="btn" href="#">See Project</a>
+        </div>
+        </section>
+      </div>
+      <!-- -------------------------------End of work section2------------------ -->
+    <!------------------------------------Work Section 3 --------------------------->
+    <div class="work-section">
+      <section >
+        <div>
+          <img class="snapShoot-portofolio" src="./images/Img/Snapshoot Portfoliotwo.png" alt="snapshoot-Portfolio-two">
+        </div>
+        <div>
+          <h1 class="owner-name">
+            Tonic
+          </h1>
+        </div>
+        <div class="info">
+            <h3>CANOPY</h3>
+            <ul>
+              <li> Back End Dev </li>
+              <li>2015</li>
+            </ul>
+          </div>
+      <div>
+        <p class="daily-selection">
+          A daily selection of privately personalized reads; no accounts or sign-ups required.
+        </p>
+      </div>
+      <div>
+        <ul class="list-of-course">
+              <li><a href="#">HTML</a></li>
+              <li><a href="#">css</a></li>
+              <li><a href="#">Javascript</a></li>
+        </ul>
+      </div>
+      <div >
+        <a  class="btn" href="#">See Project</a>
+      </div>
+      </section>
+    </div>
+
+       <!-- -------------------------------End of work section3------------------ -->
+    <!------------------------------------Work Section 4--------------------------->
+    <div class="work-section">
+      <section >
+        <div>
+          <img class="snapShoot-portofolio" src="./images/Img/Snapshoot PortfolioOne.png" alt="snapshoot-Portfolio-one">
+        </div>
+        <div>
+          <h1 class="owner-name">
+            Multi-Post <br>Stories
+          </h1>
+        </div>
+        <div class="info">
+            <h3>CANOPY</h3>
+            <ul>
+              <li> Back End Dev </li>
+              <li>2015</li>
+            </ul>
+          </div>
+      <div>
+        <p class="daily-selection">
+          A daily selection of privately personalized reads; no accounts or sign-ups required.
+        </p>
+      </div>
+      <div>
+        <ul class="list-of-course">
+              <li><a href="#">html</a></li>
+              <li><a href="#">css</a></li>
+              <li><a href="#">Javascript</a></li>
+        </ul>
+      </div>
+      <div >
+        <a  class="btn" href="#">See Project</a>
+      </div>
+      </section>
+    </div>
+  </main>
+
+  <!---------------------------------------------footer ------------------------------------------------------- -->
+  <footer>
+    <section>
+        <div class="about-section">
```

Comment:
- [ ] [REQUIRED] Good job so far! Please give this section a proper design because some `buttons` are missing and the design of the `elements` is not like [Figma](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=0-1&t=VtDtYBmX3HDlKmDq-0) 🙏🏼 

![image](https://user-images.githubusercontent.com/63915024/228899961-d66ec665-5bd9-475e-9410-977c1f41d6de.png)